images/linux/scripts/tests/DotnetSDK.Tests.ps1

```diff
@@ -0,0 +1,23 @@
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1"
+
+Describe "Dotnet" {
+
+    BeforeAll {
+        $dotnetSDKs = dotnet --list-sdks | ConvertTo-Json
+        $dotnetRuntimes = dotnet --list-runtimes | ConvertTo-Json
+        $dotnetVersions = (Get-ToolsetContent).dotnet.versions
+        $dotnet = $dotnetVersions | ForEach-Object { @{ dotnetVersion = $_ } }
+    }
+
+    It "Default Dotnet SDK is available" {
+        "dotnet --version" | Should -ReturnZeroExitCode
+    }
+
+    It "SDK $version is available" -TestCases $dotnet {
```

Comment:
This won't work.
`$dotnet` is defined in `BeforeAll`. It won't be available in `-TestCases`.
It should be located directly in Describe

Reply:
you right, fixed it